__tests__/index.spec.js

```diff
@@ -169,6 +178,19 @@ describe('#create', () => {
     expect(userObject).toMatchObject({ email: 'frodo_1@mail.com' });
   });
 
+  it('can handle references to previous attributes', () => {
+    userFabricator = buildTestUserFabricator({
+      name: ObjectFabricator.sequence(n => `Test User ${n}`),
+      email: ObjectFabricator.sequence(n => `test_user${n}@mail.com`),
+      book: ObjectFabricator.associateToMany(bookFabricator, 3),
+      author: "${name}"
+    });
+
+    const userWithAuthor = userFabricator.create();
+
+    expect(userWithAuthor.author).toEqual('Test User 1');
```

Comment:
This took me a while to understand: The 'previous' is what led me in the wrong direction. A better test description would have been e.g. 'it allows basing attribute values on other attribute values' and two attributes `name` and `author` would have been enough to make the point.

Apart from that, why did you use templates for this instead of functions (like in the sequences)? In my opinion this would lead to a more consistent API.